src/index.js

```diff
@@ -3,15 +3,19 @@ import analyticModules from './modules';
 const globalProperties = {};
 
 export const Analytics = {
-  trackView(viewName, properties) {
+  trackView(viewName, properties, modules, disabled = false ) {
+    if(disabled) return;
```

Comment:
Q: What is the point of the `disabled` parameter? If you don't want to call a function then just don't call that function 😀

Reply:
BI want to have some config file in project because they want to enable/disable events. We can add disable or use if's in project :D but... maybe we should add this feature per project...